creator-node/src/middlewares/contentAccess/contentAccessMiddleware.test.ts

```diff
@@ -89,33 +101,13 @@ describe('Test premium content middleware', () => {
     })
 
     it('returns forbidden when it fails because of invalid discovery node', async () => {
-      premiumContentAccessChecker.accessCheckReturnsWith = {
-        doesUserHaveAccess: false,
-        trackId: 1,
-        isPremium: true,
-        error: PremiumContentAccessError.INVALID_DISCOVERY_NODE
-      }
       let nextCalled = false
-      await premiumContentMiddlewareProxy.premiumContentMiddleware(
-        mockReq as unknown as Request,
-        mockRes as unknown as Response,
-        () => {
-          nextCalled = true
+      await contentAccessMiddlewareProxy({
```

Comment:
adding comment here not exactly that this line is wrong, but that some tests are now missing, since you removed the `helpers.test.ts` file.
this file should also test that it returns forbidden if CIDs don't match or if signature expired